public/beta/jspm.config.js

```diff
@@ -0,0 +1,572 @@
+SystemJS.config({
+  paths: {
+    "npm:": "jspm_packages/npm/",
+    "github:": "jspm_packages/github/"
+  },
+  browserConfig: {
+    "baseURL": "/assets",
+    "paths": {
+      "media-atom-maker/": "beta/"
+    },
+    "bundles": {
```

Comment:
As you're (correctly) not including the `bundle.js` file in the repo then we also shouldn't check in this reference here